modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/TicketResourceImpl.java

```diff
@@ -45,32 +48,50 @@ public Ticket getUserAccountPasswordResetTicket(Long userAccountId)
 	}
 
 	private Ticket _getTicket(int type, Long userAccountId) throws Exception {
-		_userLocalService.getUser(userAccountId);
+		User user = _userLocalService.getUser(userAccountId);
 
 		UserPermissionUtil.check(
 			PermissionThreadLocal.getPermissionChecker(), userAccountId,
 			ActionKeys.UPDATE);
 
-		List<com.liferay.portal.kernel.model.Ticket> tickets =
-			_ticketLocalService.getTickets(
-				User.class.getName(), userAccountId, type);
+		PasswordPolicy passwordPolicy = user.getPasswordPolicy();
 
-		if (tickets.isEmpty()) {
-			return null;
+		Date expirationDate = null;
+
+		if ((passwordPolicy != null) &&
+			(passwordPolicy.getResetTicketMaxAge() > 0)) {
```

Comment:
If this condition is not met, then the ticket is created with no `expirationDate`. This means it remains valid forever, @daiazius can you review?